tests/neuro_san/zzz_hocons/test_unit_test_hocons.py

```diff
@@ -34,7 +34,6 @@ class TestHocons(TestCase):
         "math_guy/basic_sly_data.hocon",
         "math_guy/forwarded_sly_data.hocon",
         "music_nerd/beatles_with_history.hocon",
-        "esp_decision_assistant/chicken_scenario.hocon",
```

Comment:
Remove the chicken_scenario test from the unit tests.
It moves to the integration tests (takes ~20 secs)